### DISCUSSION TITLE
What would be gained by a 351MP?

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#140] [General]
Hello one and all,

Continuing to be so incredibly impressed by the Adam image on my 350M...it is just a joy to navigate around my device...

However, VERY tempted by a mint green 351MP for £110 on sale from Droix. Got my 350M a very short amount of time before the 351MP was announced. Felt a bit dum about that...but, now having the fantastic Adam image on my 350M, what would be gained by the 351MP? Sounds like the (Emuelec?) stock firmware is more user friendly than the 350M's OpenDingux stock firmware...but that flashing 351Elec makes things much better. I gravitate to 16 bit consoles and 90s arcade games. Fun trying a few PS1 roms (I have about 200) but never used a PS1 back in the day. All I have tried seem to work fine. Maybe 351MP gets ALL PS1 working perfectly?...but my gaming focus is beneath that anyway. Sounds like above PS1 continues to be very spotty, even on the 351MP...and have just been reading several commenting on a constant audio crackle when using headphones with the 351MP, returning units, getting replacements and exact same problem. That doesn't sound good.

Whoops this got a bit long...but if anyone has bothered to read...and especially if they have experience of both Adam image on 350M AND 351MP...it doesn't seem like I will get that much more than I am already (VERY happily) experiencing by getting a 351MP?

All the best,

Chris

**Comment by eduardofilo** [comment]
Hi Chris.
My thoughs on the subject.

I don't have the RG351MP, but I do have a RG350 with the RG350M screen mod and a RG351P. I can tell you that I have not used the 351 more than to test the versions that were released from ArkOS and 351ELEC. But I guess what happens is that since there is not a huge difference between the platforms of 350 and 351 and that I have focused on the former, I feel more comfortable and I am more satisfied with adjusting the systems and/or games in 350 platform.
I also understand that the fundamental difference between both platforms is that 351 has EmulationStation and a little more power and therefore allows you to try something from 64-bit platforms and higher. But little else and in particular are platforms that interest me less.

Although I will also tell you that I believe that Adam is used mainly in RG280V, because there is no good alternative to that format on the 351 platform. If I had to start by purchasing a console with a 4: 3 screen and a 640x480 resolution, I suppose that the most logical thing would be to acquire the 351MP. But having the 350M I don't think it's worth that much.

**Comment by chrisstonehill** [comment]
Thanks for your thoughts Eduardo. I think the improvements over my 350m with Adam image will be very minimal...thus I will save some money for frighteningly high Norwegian winter electricity bills. Bad enough at the best of times (in an old, poorly insulated house)...even worse now with CRAZY high prices. 

Convinced a friend to take advantage of the deal on the 351MP though, as a massive upgrade from a well a very plastic looking, max 8bit quality...with poor emulation at that, "GameBoy" looking thing he was given for Christmas...then getting a 351MP makes a lot of sense :-D

**Comment by zugurudumba** [comment]
While the Ingenic JZ4770 is less powerful than the RK3326, it has some stellar support from the community. I've been familiar with OpenDingux since the glory days of the venerable Dingoo A320, so there are years and years of development, improvements and optimizations to profit from. My choice was obvious: RG350M.

Had the RG351MP been able to run platforms above PS1 smoothly, I might have considered it. Heck, not even the RG552 is able to do a very good job above PS1, so I'll probably upgrade when Anbernic releases something able to do Dreamcast and PSP without issues. Maybe even RG351MP or the RG552 will be able to do it after some years of firmware and software updates, but let's see how long the community stands by these devices. 351ELEC development has already ceased, so one out of 3 existing CFWs for the machine is already done for.

**Reply by chrisstonehill** [reply]
Thanks so much for these thoughts. Since applying the Adam image I have been feeling very happy to have chosen the 350M...the main reason was for the 4:3 480p screen, and the HDMI out was potentially useful (though my experience has been that it works, but is pretty unstable). DID feel a bit foolish though when barely a month after I got it the 351MP was announced. I think I picked up on the sense that it was coming, but was impatient and wanted to order right then...and my 350M came astonishingly quickly from droix.co.uk...to Norway.

Yes, I was thinking the same, that not even the RG552 seems to be that much more capable, on the stock firmware.

So most of the time I think it myself, but your words back up that I have no need to regret having gone with the 350M. Just convinced a friend in the UK to get his first (proper) retro emulating handheld in the form of the 351MP, in a Droix sale (having just been given a VERY plastic GameBoy esque thing with 400, poorly emulated, max 8bit kind of quality games on :-D ) Look forward t o hear how he gets on and hoping that he gets by on the stock firmware. I see that it has plenty of problems, but seems to be more plug and play friendly than the stock, opendingux firmware that was on my 350M.

Great that you have been involved since the Dingoo A320...is that the Grandaddy of JZ4770 devices?...from back when?

That's a shame re 351ELEC. When my friend in the UK feels brave enough to start trying to flash custom firmware (if that ever happens) I was going to suggest that. Latest sounded good...but it wont get developed any further?

**Comment by zugurudumba** [comment]
Well, I haven't been involved with development or anything like that. I just craved a Dingoo A320 back in 2009 when it launched, followed all the news about it, but could not afford one back then. The SoC inside it was the JZ4732. You can read more about it on [Wikipedia](https://en.wikipedia.org/wiki/Dingoo).

The devices coming out of China nowadays are far superior to the A320 in pretty much every way. The JZ4770 in the RG350/280 series of devices is a newer and more powerful SoC but it's very similar, making the porting of OpenDingux relatively easy. There's also a nice community for these Ingenic devices [here](https://dingoonity.org).

Of course, it seems that the era of JZ4770 devices is now over and manufacturers have moved to Rockchip SoCs (see RG351 and siblings, RG552). Maybe we'll see a more powerful Ingenic chip built into similar handhelds in the future, reigniting the development and support frenzy around OpenDingux. But for now, RG350M is the peak of a pyramid going back to 2009.

Also, it could be that we'll the manufacturers will be gravitating more and more towards Android, but it will take a while until we'll get an operating system so optimized for specific devices as OpenDingux - you wouldn't believe what people managed to run on the A320 :)

**Reply by chrisstonehill** [reply]
Interesting information. I WILL do a bit reading. Fascinating to know a bit about this little metal box of magic in my hands. Continues to blow my mind on a regular basis...ALL of it, but for example, all of the Neo Geo games. They were fabled things back in the early 90s, when I had a few games on our family Atari ST and a few more games on a 2nd hand Gameboy, that I think I bought with saved up paper round money :-) I would see screen shots in magazines that always looked incredible!...Now I can carry them all around (and so much more) in my pocket. AMAZING! My friend back in the UK just received his 351MP. With incredible will power (or the fact that he is crazy busy with work...and children) he is waiting until the weekend before he tucks in. This will be his first experience of such things. I slowly built up since autumn 2019...a SNES mini, then an AtGames handheld Megadrive thingy...pretty horrible but the ease of game library expansion with the SD card slot, then a Super Retrocade, then another one from the US so that added arcade games would actually work...and now the sublime 350M. Such a great journey! A Super Console X King (hopefully) arriving soon too. Maybe one day I'll feel like I have the time to do it a bit more from scratch and put my own system together.
